pyfar/constants/speed_of_sound.py

```diff
@@ -0,0 +1,43 @@
+"""File containing all speed of sound calculation functions."""
+import numpy as np
+
+
+def speed_of_sound_simple(temperature):
+    r"""
+    Calculate the speed of sound in air using a simplified version
+    of the ideal gas law based on the temperature.
+
+    Calculation is in accordance with ISO 9613-1 [#]_, as described in
+    ISO 17497-1 [#]_.
+
+    .. math::
+
+        c = 343.2 \cdot \sqrt{\frac{T + 273.15}{293.15}} m/s
+
+    where, T denotes the temperature in °C.
```

Comment:
```suggestion
    Calculation is in accordance with ISO 9613-1 [#]_ and  ISO 17497-1 [#]_,
    with the reference air temperature of 20°C (293.15 K).

    .. math::

        c = 343.2 \cdot \sqrt{\frac{T + 273.15}{293.15}} m/s

    where, T denotes the temperature in °C.
```

Reply:
fixed it, i did nto found the equation in 9613 before, but everthing is there.